newrelic.yml

```diff
@@ -341,7 +341,7 @@ common: &default_settings
   # embedded service within another framework and the agent is detecting the Sinatra
   # app and skipping the at_exit handler as a result. Sinatra classically runs the
   # entire application in an at_exit block and would otherwise misbehave if the
-  # Agent's at_exit handler was also installed in those circumstances. Note:
+  # agent's at_exit handler was also installed in those circumstances. Note:
```

Comment:
The `newrelic.yml` file is autogenerated by the `newrelic:update_newrelicyml` rake task. The descriptions for the config values originate in [`lib/new_relic/agent/configuration/default_source.rb`](https://github.com/newrelic/newrelic-ruby-agent/blob/dev/lib/new_relic/agent/configuration/default_source.rb). I don't believe that file was updated as part of this PR. Could you update the `force_install_exit_handler` description in that file?

Reply:
updated this one -- thanks for pointing these ones out that I missed!